core/src/main.cpp

```diff
@@ -847,6 +846,36 @@ auto load_oidc_configuration(const json& _config, json& _oi_config, json& _endpo
 	return true;
 } // load_oidc_configuration
 
+auto load_user_mapping_plugin(const json& _user_mapping_config) -> bool
+{
+	const auto& map_lib{_user_mapping_config.at("plugin_path").get_ref<const std::string&>()};
+	const auto config_string{to_string(_user_mapping_config.at("configuration"))};
+
+	// Load shared lib
+	logging::trace("{}: Loading shared library at [{}]", __func__, map_lib);
```

Comment:
Missing period at the end of all new log statements.

Reply:
Believe I got them all.